nym-api/src/coconut/api_routes/mod.rs

```diff
@@ -94,7 +52,16 @@ pub async fn post_free_pass(
     debug!("Received free pass sign request");
     trace!("body: {:?}", freepass_request_body);
 
-    validate_freepass_public_attributes(&freepass_request_body)?;
+    //check expiration date validity
+    if freepass_request_body.expiration_date > freepass_exp_date_timestamp() {
+        return Err(CoconutError::TooLongFreePass {
+            expiry_date: OffsetDateTime::from_unix_timestamp(
+                //safety : expiration date is a unix timestamp so these unwraps will not fail for 290 million years
+                freepass_request_body.expiration_date.try_into().unwrap(),
```

Comment:
absolutely not. the request comes from an user with no sanitisation. so a malicious client could crash the nym-api by providing an invalid timestamp.

Reply:
Fixed in fa1cf7106c7a178fad8d11a99d3ca0a15e09376a